librocksdb_sys/crocksdb/c.cc

```diff
@@ -3445,6 +3450,41 @@ void crocksdb_ingest_external_file_cf(
   SaveError(errptr, db->rep->IngestExternalFile(handle->rep, files, opt->rep));
 }
 
+bool crocksdb_ingest_external_file_optimized(
```

Comment:
Can we test if this optimized version really take effect in the real world?

Reply:
So how about add a log or metrics here to show it fallbacked or not?

Reply:
Maybe benchmark it, and see if write stall duration reduce?

Reply:
Yeah, but I think we should better do it when we integrate this into TiKV.